io.github.letslearn-app.yaml

```
@@ -0,0 +1,55 @@
+app-id: io.github.letslearn-app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
```

```suggestion
  - org.freedesktop.Sdk.Extension.node16
```

and the other path must be updated too.